common/buildcraft/core/BCCoreConfig.java

```diff
@@ -177,6 +179,11 @@ public static void preInit(File cfgFolder) {
         propMarkerMaxDistance.setComment("How far, in minecraft blocks, should markers (volume and path) reach?");
         none.setTo(propMarkerMaxDistance);
 
+        propPumpMaxDistance = config.get(general, "pumpMaxDistance", 64);
+        propPumpMaxDistance.setMinValue(16).setMaxValue(256);
+        propPumpMaxDistance.setComment("How far, in minecraft blocks, should pumps reach in water?");
```

Comment:
Probably change comment to "How far, in minecraft blocks, should pumps reach in fluids?"